vnext/ReactWindowsCore/Modules/PlatformConstantsModule.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+
+#include <cxxreact/CxxModule.h>
+
+namespace facebook::react {
+
+class PlatformConstantsModule : public facebook::xplat::module::CxxModule {
+ public:
+  static const char *name;
```

Comment:
>n [](start = 21, length = 1)

We are getting a lot of warnings when fields use the same naming conventions as local variables. Consider to add a prefix use Pascal casing 'Name'.

Reply:
Can do this, along with updating the others already using lowercase.